backend/app/api/routes/doc_transformation_job.py

```
@@ -0,0 +1,45 @@
+    crud = DocTransformationJobCrud(session)
+    job = crud.read_one(job_id)
+    return APIResponse.success_response(job)
+
+@router.get(
+    "/",
+    description="Get the status and details of multiple document transformation jobs by IDs.",
+    response_model=APIResponse,
+)
+def get_multiple_transformation_jobs(
+    session: SessionDep,
+    current_user: CurrentUser,
```

Ruff (0.12.2)

Unused function argument: `current_user`

(ARG001)